### DISCUSSION TITLE
Slideshow for images

### DISCUSSION THREAD

**Discussion by PeterHoeche** [outdoorbits/little-backup-box#307] [Ideas]
I recently started using LBB and wanted to see a preview of some of the images. It's a bit cumbersome to click through the images. It might be good to integrate some kind of slideshow or lightbox.

**Comment by outdoorbits** [comment]
Nice idea, shouldn't be a big deal. I have it on my to-do list but there is a bit in the queue before...

**Comment by outdoorbits** [comment]
It's online, you'll find it into the single view. Please test it.

**Reply by PeterHoeche** [reply]
Thanks a lot. I have tested it and discovered a few things.

1. The update via the web-ui didn't work for me.
It seems to be related to my setup on the raspberry pi. I used `Raspberry Pi Imager` for the initial setup and also created a different user than `pi`. 
Here's the log:
```
sh: 1: cd: can't cd to ~pi
sudo: unknown user pi
sudo: error initializing audit plugin sudoers_audit
curl: (23) Failure writing output to destination
```

2. The slideshow doesn't work as intended for me.
When I click on any image and start the slideshow, it always jumps to the last imported image and keeps refreshing the page in slideshow interval. 

3. The slideshow doesn't work on a small screen.
I have a raspberry pi with a touch screen and when I start the slideshow the image is not visible (only the filter and the page overview are visible).

**Comment by outdoorbits** [comment]
1. For web based update you need user pi. Please just create it or change your imager settings.
2. I had the same issue after it worked before??? It's fixed now.
3. I removed the headline to save space. Do you have a solution for presenting full screen images?

**Reply by outdoorbits** [reply]
... I found a solution. It's full screen now.

**Reply by PeterHoeche** [reply]
Thank you very much! I have tested it and it works as expected.
